cassandra-sink/src/main/java/org/springframework/xd/sink/cassandra/CassandraSinkConfiguration.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.xd.sink.cassandra;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cassandra.core.ConsistencyLevel;
+import org.springframework.cassandra.core.RetryPolicy;
+import org.springframework.cassandra.core.WriteOptions;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.data.cassandra.core.CassandraOperations;
+import org.springframework.expression.Expression;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+import org.springframework.integration.annotation.ServiceActivator;
+import org.springframework.integration.cassandra.outbound.CassandraMessageHandler;
+import org.springframework.integration.channel.DirectChannel;
+import org.springframework.integration.config.EnableIntegration;
+import org.springframework.messaging.MessageChannel;
+import org.springframework.messaging.MessageHandler;
+import org.springframework.util.StringUtils;
+import org.springframework.xd.module.cassandra.CassandraConfiguration;
+
+/**
+ * @author Artem Bilan
+ */
+@Configuration
+@EnableIntegration
+@Import(CassandraConfiguration.class)
+public class CassandraSinkConfiguration {
+
+	private static final SpelExpressionParser PARSER = new SpelExpressionParser();
+
+	@Value("#{environment.consistencyLevel}")
```

Comment:
If there is no `consistencyLevel` in the `PropertySource`s we end up with   
```
Could not autowire field: private org.springframework.cassandra.core.ConsistencyLevel org.springframework.xd.sink.cassandra.CassandraSinkConfiguration.consistencyLevel; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'consistencyLevel' in string value "${consistencyLevel}"
```
if we just use PP style here (`@Value("${consistencyLevel}")`) instead of this SpEL style against `environment` bean which can return `null` for us.

From other side I guess XD PP Configurer can be configured with `setIgnoreUnresolvablePlaceholders(false)`.